python/pyspark/sql/pandas/group_ops.py

```diff
@@ -635,7 +635,7 @@ def __transformWithState(
         from pyspark.sql import GroupedData
         from pyspark.sql.functions import pandas_udf
         from pyspark.sql.streaming.stateful_processor_util import (
-            TransformWithStateInPySparkUdfUtils,
+            TransformWithStateInPandasUdfUtils,
```

Comment:
Is this name user-facing? Another option is we do this rename in 4.0 if the original name is very confusing.

Reply:
It's not user facing - it just got added into the serialization form of python function to be included in the plan.

I think this is not a huge deal as long as we have comment on the rationale.

Reply:
ah I see, then we are fine.